### PR TITLE
SEIaImIsRD vignette edits

### PR DESCRIPTION
**Pull request by solveigvandervegt**
Removed waning immunity from this model.

Edited vignette

## Issue
- [ ] Which issue does your pull request address? If none of the existing issues are applicable, you can write a new issue.

## Testing, documentation, and checks
- [ ] If your pull request introduces new code, you should write automated tests (explained [here](https://r-pkgs.org/tests.html)).
- [ ] Check that all tests pass using `devtools::check()`.
- [ ] Check compliance with the preferred R style using `lintr::lint_package()`.
- [ ] Build documentation using `devtools::document()`.
- [ ] If you changed the readme, run `knitr::knit("README.Rmd")`.

## Pull request
- [ ] Assign at least one reviewer.

### CODE REVIEW

File: vignettes/SEIRD_symptom_compartments.Rmd [hunk 1]
```diff
@@ -123,109 +106,103 @@ initial_conditions(model) <- list(S = S, E = E, asymptomatic = asymptomatic,
                                   mild = mild, severe = severe, R = R, D = D)
```
Comment (hfldai) [inline vignettes/SEIRD_symptom_compartments.Rmd]: Should be ```initial_conditions(model) <- list(S = S, E = E, I_asymptomatic = asymptomatic, I_mild = mild, I_severe = severe, R = R, D = D)```

**Review by hfldai — COMMENTED** [review]
Many thanks for the edits. I think everything is fine. The only minor issue is that the names of the infectious population groups when setting the initial conditions in the vignette were not consistent with the model (xx -> I_xx). 

Also I see some edits on the basic reproduction number. Is there something wrong or just with the namings?

File: vignettes/SEIRD_symptom_compartments.Rmd (hunk 1 above)
Comment (solveigvandervegt) [reply vignettes/SEIRD_symptom_compartments.Rmd]: Feel free to change that if you want! Ben just asked me to go over the vignette so it is ready for publication. We had a bit of a back and forth about R0 but in the end I just changed the names in the class cause I think F might be a protected name in R. It shows up in a different color than other variables if you just call the matrix F so I though it safer to rename slightly

**Review by ben18785 — APPROVED** [review]
Looks good! Going to merge in now.

**Pull request merged**